spec/random-data-spec.coffee

```diff
@@ -279,3 +279,9 @@ describe "Random Data", ->
   it "inserts random tv", ->
     spyOn(chance, 'tv').andReturn('KICE')
     dataTest 'tv', 'KICE'
+  it "inserts random longitude", ->
+    spyOn(chance, 'longitude').andReturn('-49.273096')
```

Comment:
I think `spyOn(change, 'longitude')` should probably be `spyOn(change, 'float')`